File: app_java_ui/src/main/java/ru/rbs/mobile/payment/sample/java/MarketApplication.java
package ru.rbs.mobile.payment.sample.java;

import android.app.Application;

import ru.rbs.mobile.payment.sdk.SDKPayment;

public class MarketApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        SDKPayment.INSTANCE.init(this, null);
    }
}

File: app_kotlin_ui/src/main/kotlin/ru/rbs/mobile/payment/sample/kotlin/MarketApplication.kt
@file:Suppress("UndocumentedPublicClass")
package ru.rbs.mobile.payment.sample.kotlin

import android.app.Application
import ru.rbs.mobile.payment.sdk.SDKPayment

@Suppress( "UndocumentedPublicClass")
class MarketApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        SDKPayment.init(this)
    }
}
